core/admin/mailu/models.py

```diff
@@ -260,13 +260,12 @@ def resolve_domain(cls, email):
 
     @classmethod
     def resolve_destination(cls, localpart, domain_name, ignore_forward_keep=False):
-        localpart_stripped = None
+        localpart_maybe_split = localpart
```

Comment:
This breaks the user query in line 274. I would suggest to leave the name of the variable as `localpart_stripped`.